test/autobahn/server.ml

```diff
@@ -1,20 +1,25 @@
-open Riot
```

Comment:
As a rule of thumb, we tend to open Riot at the top of every file, to avoid having to open it in other places.